docs/samples/v1beta1/torchserve/README.md

```diff
@@ -0,0 +1,130 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Build image transformer
+
+This example requires image transformer for converting bytes array image to json array. Refer torchserve image tranformer for building image.
+
+[Image-transformer](../../transformer/torchserve_image_transformer/README.md)
+
+## Creating model storage with model archive file
```

Comment:
Let's also add the torchserve model archive [reference](https://github.com/pytorch/serve/blob/master/model-archiver/README.md) as users may not be familiar with the concept.

Reply:
I have uploaded the model store to `gs://kfserving-examples/models/torchserve/image_classifier` which contains `mnist.mar` and `config.properties`, we can also use remote storage as alternative way instead of  PVC which might be too many steps for a user to run this example.